xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,487 @@
+         when all commands finish successfully (return code is zero), the heuristics
+         has passed.
+      </para>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-status">
+      <title>Checking and Showing Quorum Status</title>
+      <!-- HINT:
+The Ring ID and some other parts are numbers which are actually an IP address.
+You can "decrypt" it by using Python:
+
+$ python3
+>>> import ipaddress
+# Convert a number to an IP address:
+>>> ipaddress.ip_address(3232235778)
+IPv4Address('192.168.1.2')
+# Convert an IP address to a number:
+>>> int(ipaddress.IPv4Address("192.168.1.2"))
+3232235778
+      -->
+      <para>
+         To assess the quorum of your cluster, you can do so on one
+         of your cluster nodes as shown in <xref
+            linkend="ex-ha-qdevice-crm-corosync-status-quorum"/>.
+         It shows you the status of your &qdevice; components.
+      </para>
+
+      <example xml:id="ex-ha-qdevice-crm-corosync-status-quorum">
+         <title>Output Status of &qdevice;</title>
+         <screen>&prompt.root;<command>crm</command> corosync status quorum <co xml:id="co-quorum-cmd"/>
+ Quorum information
+------------------
+Date:             ...
+Quorum provider:  corosync_votequorum
+Nodes:            2 <co xml:id="co-quorum-nodesnumber"/>
+Node ID:          3232235777 <co xml:id="co-quorum-nodeid"/>
+Ring ID:          3232235777/8
+Quorate:          Yes <co xml:id="co-quorum-quorate"/>
+
+Votequorum information
+----------------------
+Expected votes:   3
+Highest expected: 3
+Total votes:      3
+Quorum:           2
+Flags:            Quorate Qdevice
+
+Membership information
+----------------------
+    Nodeid      Votes    Qdevice Name
+ 3232235777         1    A,V,NMW &subnetI;.1 (local) <co xml:id="co-quorum-nodes"/>
+ 3232235778         1    A,V,NMW &subnetI;.2 <xref linkend="co-quorum-nodes"/>
+         0          1            Qdevice</screen>
+         <calloutlist>
+            <callout arearefs="co-quorum-cmd">
+               <para>
+                  You can also use the command <command>corosync-quorumtool</command>
+                  as an alias with an identical result.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-nodesnumber">
+               <para>
+                  The number of nodes we are expecting. In this example, it is a
+                  two-node cluster.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-nodeid">
+               <para>
+                  The node ID as an integer value. This ID is an integer representation
+                  of the IP address. In this example, the value <literal>3232235777</literal>
+                  stands for the IP address <systemitem
+                     class="ipaddress">&subnetI;.1</systemitem>.
+                  <remark>toms 2020-05-12: I'm looking for a specific term of
+                  this integer value to represent the IP address. Any hints?
```

Comment:
Ok, great. With your previous comment and this recommendation, the paragraph looks like:

```xml
In this case, since it is not explicitly specified in 
<filename>corosync.conf</filename>, this ID is an 32-bit integer
representation of the IP address. In this example, the value
<literal>3232235777</literal> stands for the IP address <systemitem
 class="ipaddress">&subnetI;.1</systemitem>.
```